pkg/api/routes_test.go

```diff
@@ -1134,7 +1135,7 @@ func TestRoutes(t *testing.T) {
 
 			status = testUpdateBlobUpload(
 				[]struct{ k, v string }{
```

Comment:
Are you sure we can hardcode these digests? In the past we've had problems since different skopeo versions on different developer environments did the conversion to oci format differently, resulting in different hashes.

Reply:
I think in these tests the digest is just formally put there for the digest check, it's not actually used. The image store is being mocked. I tried running with changes to the digest and it passes.